ct.yaml

```diff
@@ -0,0 +1,6 @@
+# See https://github.com/helm/chart-testing#configuration
+remote: origin
+chart-dirs:
+  - chart
+chart-repos: []
+helm-extra-args: --timeout 600
```

Comment:
Why do we need this?

Reply:
The same reason as above: The directory is named "chart" instead of "charts". Not pretty sure if we need the helm-timeout...
This file is picked up automatically from chart-testing-action.